manifests/a/AAS/WorldWideTelescope/6.1.1/AAS.WorldWideTelescope.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.5-1-19041-1682
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: AAS.WorldWideTelescope
+PackageVersion: 6.1.1
+PackageLocale: en-US
+Publisher: WorldWide Telescope
+PublisherUrl: https://aas.org/
+# PublisherSupportUrl: 
+PrivacyUrl: http://worldwidetelescope.org/terms/
+# Author: 
+PackageName: WorldWide Telescope
```

```suggestion
PackageName: AAS WorldWide Telescope
```
This must match control panel